### PR TITLE
Added Slider Component

### PR DESCRIPTION
**Pull request by karthiknmenon**
@goutham-subramanyam _a Please review.

### CODE REVIEW

File: lib/components/Slider.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  labelProps,
+  required,
+  helpText,
+  min,
+  max,
+  value,
+  stepSize,
+  labelStepSize,
+  className,
+  onChange}) => {
```
Comment (goutham-subramanyam) [inline lib/components/Slider.js:18]: All these props except classname can be passed to slider as otherProps.

**Review by goutham-subramanyam — COMMENTED** [review]
@karthiknmenon _a

File: lib/components/Slider.js (hunk 1 above)
Comment (karthiknmenon) [reply lib/components/Slider.js]: Done

**Comment by karthiknmenon** [comment]
@goutham-subramanyam _a I have made the required changes. Please review.

**Review by goutham-subramanyam — APPROVED** [review]
LGTM. Merging.

**Pull request merged**